README.md

```diff
@@ -10,10 +10,10 @@ Uranium is released under the terms of the LGPLv3 or higher. A copy of this lice
 
 Dependencies
 ------------
-- Qt 5 with QtQuick 2, QtQuick Controls
+- [Qt 5](https://doc.qt.io/qt-5/) with [QtQuick 2](https://doc.qt.io/archives/qt-5.11/qtquickcontrols2-index.html), [QtQuick Controls](https://doc.qt.io/qt-5/qtquickcontrols-index.html)
```

Comment:
Please refer to the correct version of Qt: 5.10. But actually I think that the links to QtQuick and QtQuick Controls can be removed. It's an incomplete list of the Qt modules we're using anyway.